recipe/install.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ "${PKG_NAME}" == "librdkit" ]; then
+
+    # NOTE(skearnes): List is from `make list_install_components`, excluding "python" and "pgsql".
+    for COMPONENT in Unspecified base data dev docs extras runtime; do
```

Comment:
```suggestion
    for COMPONENT in Unspecified base data dev extras runtime; do
```

Let's review which components we want to throw in `librdkit`. For example, `docs` might belong elsewhere.

Reply:
`docs` should probably be dropped and just found in the github source; let me double check on `dev` and see if it makes sense to move that to the librdkit-dev output (and make that not just windows-only)